assets/airports/airportLoadList.js

```diff
@@ -424,6 +424,12 @@ window.AIRPORT_LOAD_LIST = (function() { // eslint-disable-line wrap-iife
             level: 'hard',
             name: 'Shanghai Pudong International Airport',
             wip: true
-        }
+        },
+		{
+			icao: 'kcvg',
+			level: 'easy',
+			name: 'Cincinnati-Northern Kentucky International Airport',
```

Comment:
you could probably shorten `International` to `Intl` so this doesn't wrap to a second line